src/Piston/scss/base.scss

```diff
@@ -6,6 +6,16 @@
 
 ========================================================================== */
 
+// Override base's default variables
+$base-font-family: 'Bitter', serif; 
+$base-heading-font-family: 'Raleway', sans-serif;
+$base-line-height: 30;
+$base-link-color: black;
+$base-link-hover-color: #808080;
+$banner-background-color: black;
+$brand-font-color: white;
+$banner-font-color: white;
```

Comment:
It was the default folder structure from the base download so I left it. I think we should move all the ad-hoc styling from site.scss into individual files to make it easier to manage, will make an issue for that.